samples/zigbee/light_switch/src/main.c

```diff
@@ -303,6 +305,12 @@ static void find_light_bulb_cb(zb_bufid_t bufid)
 	}
 }
 
+static void find_light_bulb_alarm(struct k_timer *timer)
```

Comment:
Do we really need this additional step? Why not just modify ```find_light_bulb``` parameters and call it directly?

Reply:
Because it is necessary to call find_light_bulb function from ZBOSS context. So we need use ZBOSS schedule callbacks.

Reply:
But why? It shouldn't be necessary to do that.

Reply:
I've removed this step.

Reply:
Now zb_buf_get_out_delayed() function is thread safe - fix in PR: nrfconnect/sdk-nrfxlib#214.
But this step is necessary for allocate new buffer for zb_zdo_match_desc_req() in find_light_bulb function.